allen-cell-animated/cell-catalog
# Feature/genomic characterization data

Problem
=======
What is the problem this work solves, including
closes #49 

Solution
========
What I/we did to solve this problem
added fields for genomic characterization data in admin config file
with @meganrm 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. `npm run`
2. `npx netlify-cms-proxy-server`
3. check http://localhost:8000/admin/#/collections/disease-cell-line/entries/AICS-104/index

or
https://deploy-preview-54--cell-catalog.netlify.app/admin/#/collections/disease-cell-line/entries/AICS-104/index

Screenshots (optional):
-----------------------
Show-n-tell images/animations here
<img width="834" alt="Screenshot 2024-09-05 at 2 51 21 PM" src="https://github.com/user-attachments/assets/97505ebd-5ae5-4a52-b801-9c2e77343bc7">
